dev/ci/gitlab-modes/tagged-runners.yml

```diff
@@ -1,3 +1,11 @@
 .auto-use-tags:
   tags:
     - $TAGGED_RUNNERS
+
+# making this configurable is too annoying if we want to support > 1 tag
+# (when I tried medium ci.inria.fr it timed out after 1h, large finished in 30min)
```

Comment:
actually this is nonsense, what really happened was that the attempt with medium managed to build the base image and timed out during the edge image, then the attempt with large got the prebuilt base image.